Python/symtable.c

```diff
@@ -940,6 +942,12 @@ drop_class_free(PySTEntryObject *ste, PyObject *free)
         return 0;
     if (res)
         ste->ste_needs_classdict = 1;
+    res = PySet_Discard(free, &_Py_ID(__conditional_annotations__));
+    if (res < 0)
+        return 0;
+    if (res) {
+        ste->ste_has_conditional_annotations = 1;
+    }
```

Comment:
Do you want to mix PEP-7 or not? maybe we could remove the `{}` in `if (res)` 

```suggestion
    if (res)
        ste->ste_has_conditional_annotations = 1;
```

Because I think it's better to be non PEP-7 here rather than stick to it (at least in this specific function)

Reply:
I'd prefer to add braces around any new conditionals.